share/container-build.sh

```diff
@@ -2,17 +2,13 @@
 
 #
 # SPDX-FileCopyrightText:  2023, Iker Pedrosa <ipedrosa@redhat.com>
+# SPDX-FileCopyrightText:  2024, Iker Pedrosa <ipedrosa@redhat.com>
 #
 # SPDX-License-Identifier:  BSD-3-Clause
 #
 
-for FILE in share/containers/*; do
-	IFS='/'
-	read -ra ADDR <<< "$FILE"
-	IFS='.'
-	read -ra ADDR <<< "${ADDR[2]}"
-	IFS=''
-	if ! docker build -f $FILE . --output build-out/${ADDR[0]}; then
-		exit
-	fi
-done
+pushd share/ansible/ || exit
+ansible-playbook playbook.yml -i inventory.ini -e 'distribution=alpine'
+ansible-playbook playbook.yml -i inventory.ini -e 'distribution=debian'
+ansible-playbook playbook.yml -i inventory.ini -e 'distribution=fedora'
+popd || exit
```

Comment:
Do we want to `set -eE` instead of `|| exit`?  Also, what if ansible fails?

Reply:
Yes, that would make more sense